### PR TITLE
Modify 3PlayMedia turnaround levels

### PR DESCRIPTION
**Pull request by mushtaqak**
[EDUCATOR-1809](https://openedx.atlassian.net/browse/EDUCATOR-1809)

Related VAL PR: https://github.com/edx/edx-val/pull/111
Related platform PR: https://github.com/edx/edx-platform/pull/16672

[Sandbox](https://studio-pr16672.sandbox.edx.org/videos/course-v1:edX+DemoX+Demo_Course)

### CODE REVIEW

**Comment by mushtaqak** [comment]
@muhammad-ammar @Qubad786 This is ready for your review

File: VEDA_OS01/models.py [hunk 1]
```diff
@@ -85,18 +85,20 @@ class ThreePlayTurnaround(object):
     Turnaround Enumeration.
     Its the time taken by 3PlayMedia transcription process.
     """
-    EXTENDED_SERVICE = 'extended_service'
+    EXTENDED = 'extended'
     DEFAULT = 'default'
```
Comment (Qubad786) [inline VEDA_OS01/models.py]: defaults to standard according to docs

File: VEDA_OS01/models.py [hunk 2]
```diff
@@ -85,18 +85,20 @@ class ThreePlayTurnaround(object):
...
-    SAME_DAY_SERVICE = 'same_day_service'
+    EXPEDITED = 'expedited'
+    RUSH = 'rush'
+    SAME_DAY = 'same_day'
+    TWO_HOUR = 'two_hour'
 
     CHOICES = (
-        (EXTENDED_SERVICE, '10-Day/Extended'),
+        (EXTENDED, '10-Day/Extended'),
         (DEFAULT, '4-Day/Default'),
```
Comment (Qubad786) [inline VEDA_OS01/models.py]: defaults to standard

**Comment by mushtaqak** [comment]
@Qubad786 Addressed feedback

**Pull request merged**